test/unit/spec/support/supported_platforms.rb

```diff
@@ -1,6 +1,6 @@
 def supported_platforms
   {
     'ubuntu' => ['14.04', '16.04', '18.04'],
-    'centos' => ['6.10', '7.6.1804'],
+    'centos' => ['6.10', '7.6.1810'],
```

Comment:
According to https://github.com/chefspec/fauxhai/blob/master/PLATFORMS.md, `7.6.1804` is deprecated now, so I bumped to next supported